haystack/pipeline_eval.py

```diff
@@ -0,0 +1,99 @@
+from haystack.document_store.elasticsearch import ElasticsearchDocumentStore
```

Comment:
Do we need this script here pipeline_eval.py? We should only have the tutorial and test, right?

Reply:
Now deleted